### PR TITLE
add no logic weekly preset

### PR DESCRIPTION
**Pull request by shirosoluna**
Add the no logic "weekly" preset that has been used for the last several months.

Notable highlights:

No logic
4 Dungeon Reward Bridge Spawn Condition
7 Dungeon Reward Ganon's Boss Key 
4 shops, random prices + shuffled merchants
Small Keysanity Anywhere
Boss Keys Regional 
Random Age
Affordable Scrubs
Dungeon Entrances + Ganon's Castle Randomized
Chest textures match content - with shields, bombchus, and upgrades in major chests
Strong Hints + Magic 
Random cuccos
Bombchu bag and drops enabled
Blue Fire Arrows enabled
Multiple Misc hints for skull house, hookshot, frogs
No disabled locations

### CODE REVIEW

File: .vscode/settings.json
Comment (fenhl) [inline .vscode/settings.json:1]: Please remove this file.

**Review by fenhl — CHANGES_REQUESTED** [review]

File: .vscode/settings.json
Comment (shirosoluna) [reply .vscode/settings.json:1]: deleted, hopefully done correctly

**Comment by fenhl** [comment]
Before this can be fully reviewed, please take care of the failing CI checks first.

**Comment by shirosoluna** [comment]
sorry, I dont have python locally to check this so just doing this in github web for now which has its pros/cons
should be resolved for the 3 issues that I saw from the CI checks

**Review by fenhl — APPROVED** [review]

**Pull request merged**